### PR TITLE
Fix/#72 Library carousel not responsive

### LINKED ISSUE #72: Library carousel not responsive
**Describe the problem**
When in library view on mobile screens, the carousel component makes the page to display in full view. 

**Actual behavior**
The carousel component makes the page display full view

**Expected behavior**
The carousel component will shrink in width on mobile screens

**To Reproduce**
1. Go to [Library View](https://www.dekoo.tk/library)
2. Open developer tools 
3. Click on the responsive tab
4. Select any mobile device 
5. See error

Alternatively open [Library View](https://www.dekoo.tk/library) on any mobile device

**Screenshots**

**Your Environment:**

**Additional Information**
I did not test this, or look into the problem but my initial thought is that the carusel which is using [react-slick](https://react-slick.neostack.com/) is having a fixed width and mobile devices are taking that width as page width.

Initially I wanted to cut on dev time and reuse for it [react-slick](https://react-slick.neostack.com/) but if seen fit, we can drop the `react-slick` implementation and do a basic carousel some other way.

### PR DESCRIPTION
**Pull request by thomasgilmore**
**Task Description**
Makes the library carousels responsive 

Resolves #72

### CODE REVIEW

**Review by Kerosz — CHANGES_REQUESTED** [review]
I've noticed that now it has too much space between the `cards`. I think that's why I added all those `width` calculations there, to not have that annoying spacing between `cards` when you resize.

image

Also if you add another dataset to that categories data, that only has 2 books in it, this is the displayed result

image

This is the dataset I've added to get the above result:

`data/books.ts`
```js
  {
    category: 'Saved Books',
    link: '/saved',
    data: [
      {
        name: 'Lean Project Management Explained',
        author: 'Can Akdeniz',
        src: 'https://imgv2-2-f.scribdassets.com/img/audiobook_square_badge/399285183/original/216x216/ade11cc8e3/1629025589?v=1',
        rating: 4,
      },
      {
        name: 'Your Next Five Moves: Master the Art of Business Strategy',
        author: 'Patrick Bet-David',
        src: 'https://imgv2-1-f.scribdassets.com/img/audiobook_square_badge/445801005/original/216x216/2c6db4dc9c/1630690492?v=1',
        rating: 5,
      },
    ],
  },
  ```

File: src/components/library/carousel.tsx [hunk 1]
```diff
@@ -27,8 +27,8 @@ export default function Carousel({
 }: ICarousel) {
   const slidesToShow = 6;
   const totalCardsToShow = slidesToShow <= data.length ? slidesToShow : data.length;
-  const widthOffset = data.length > slidesToShow ? 174 : 20 * totalCardsToShow;
-  const totalCarouselWidth = totalCardsToShow * BASE_CARD_WIDTH + widthOffset;
+  // const widthOffset = data.length > slidesToShow ? 174 : 20 * totalCardsToShow;
```
[the remark this answers is not in the document]
Comment (thomasgilmore95) [reply src/components/library/carousel.tsx:30]: Ok

**Comment by Kerosz** [comment]
@all-contributors please add @thomasgilmore95 for code

**Comment by Kerosz** [comment]
Closing this as the work has been moved to this PR #79

**Pull request closed without merging**